tests/test_cli.py

```diff
@@ -350,25 +350,33 @@ def test_datasets_import(data_file, data_repository, runner, project, client):
     assert result.exit_code == 0
 
 
-def test_datasets_list_empty(runner, project):
+def test_datasets_list_empty(runner):
     """Test listing without datasets."""
     result = runner.invoke(cli.cli, ['dataset'])
     assert result.exit_code == 0
 
-    rows = result.output.split('\n')
-    assert len(rows) == 3
+    output = result.output.split('\n')
+    assert set(output.pop(0).split()) == {'AUTHORS', 'CREATED', 'ID', 'NAME'}
```

Comment:
with https://github.com/SwissDataScienceCenter/renku-python/pull/425/files#r259710131 you can keep using `list`